tests/lib/views/new_recipe/view.js

```
@@ -0,0 +1,17 @@
+var View = require('../view');
+var NewRecipeAccessors = require('./accessors');
+
+
+function NewRecipeView() {
+  [].push.call(arguments, NewRecipeAccessors);
+  View.apply(this, arguments);
+
+  this.accessors.newRecipePage; // Wait until it appears
```

This line should fail, at the moment `newRecipePage` is not defined.